build.gradle

```diff
@@ -360,3 +360,32 @@ uploadArchives {
         System.out.println("Uploading version $version")
     }
 }
+
+ext.htmlDir = new File("build/docs/html")
+
+//update static web docs
+task copyJavadoc(dependsOn: 'javadoc', type: Copy) {
+    from 'build/docs/javadoc'
+    into "$htmlDir/javadoc"
+}
+
+task copyPicardDoc(dependsOn: 'picardDoc', type: Copy){
+    from 'build/docs/picarddoc'
+    into "$htmlDir/picarddoc"
+}
+
+task updateGhPages(dependsOn: ['copyJavadoc', 'copyPicardDoc']){
```

Comment:
Would it be too much to spell out `github` here? I didn't know what `gh` was until I got to line 383.